features/integration/abi.feature

```diff
@@ -28,7 +28,7 @@ Feature: ABI Interaction
     When I create the Method object from method signature "<method-signature>"
     And I create a new method arguments array.
     And I append the encoded arguments "<app-args>" to the method arguments array.
-    And I add a method call with the transient account, the current application, suggested params, on complete "noop", current transaction signer, current method arguments.
+    And I add a hash-deterministic method call with the transient account, the current application, suggested params, on complete "noop", current transaction signer, current method arguments.
```

Comment:
All pre-existing method calls in cucumber tests are `hash-deterministic` but the C2C tests need a `hah-nondeterministic` variant. This will require refactoring the `I add method call...` steps (there are 3 variants)